back-end/API/app/controllers/enterprise_controller.py

```
@@ -38,33 +38,34 @@ def getEnterpriseByEmployer(id):
             data["id"] = enterprise.id
             if enterprise and enterprise.id == data["id"]:
                 enterprise_service.updateEnterprise(data)
                 return jsonify({'message': 'enterprise updated'})
             else:
                 logger.warn("An user tried to modify an entreprise don't have permission")
                 return jsonify({'message': 'User cannot modify enterprise'}), 401
-
-        
-        logger.warn(f'Enterprise not found with id : {id}')
-        return jsonify({'message': 'enterprise not found'}), 404
     except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)